README.md

```diff
@@ -45,5 +45,25 @@ gulp test/dotnet
 2. Next, implement the ```run``` method under the rule that actually does the validation. Add a reference to this script file under ```src/typescript/azure-openapi-validator/index.ts```.
 3. Lastly add a test case for the validation rule, by adding a test json/yaml under ```src/typescript/azure-openapi-validator/tests/resources``` and a script under ```src/typescript/azure-openapi-validator/tests``` depending on the type of the validation rule.
 
+## Changelog
+### What's New (07/07/2017)
+Resolved issues
+-   https://github.com/Azure/autorest/issues/2305 
+-   https://github.com/Azure/autorest/issues/2389 
+Enhancements
+-	New validation rule for Post operationIds 

```

Comment:
For new rules, could you add a link to the documentation? I'd like to have information here, so that if we send it to partners they get good context on what to expect next time they run the validator.